libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -511,18 +529,44 @@ class MapboxNavigation(
          * Use [NavigationOptions.toBuilder] to easily customize selected options.
          */
         @JvmStatic
-        fun defaultNavigationOptions(context: Context): NavigationOptions {
-            return NavigationOptions.Builder(
-                NONE_SPECIFIED,
-                ROUNDING_INCREMENT_FIFTY,
-                DEFAULT_NAVIGATOR_POLLING_DELAY,
-                MapboxDistanceFormatter(
-                    context.applicationContext,
-                    null,
-                    UNDEFINED,
-                    ROUNDING_INCREMENT_FIFTY
+        fun defaultNavigationOptions(context: Context, accessToken: String?): NavigationOptions {
+            val tilesUri = URI("https://api-routing-tiles-staging.tilestream.net")
+            val tilesVersion = "2019_04_13-00_00_11"
```

Comment:
Latest version available for `tilesUri` is `2020_02_02-03_00_00`, `tilesUri` is also a staging endpoint, do we want to offer these values as default? It seems that these will become out of date quickly and we'll need to update the code accordingly, so wondering if it's worth exposing 🤔

I believe that was the reason why @kmadsen (correct me if not) wanted to strip that out of `MapboxNavigation` https://github.com/mapbox/mapbox-navigation-android/pull/2480/files#r379545433 I know that we'd be exposing `defaultNavigationOptions` but not sure if it's worth it as they're going to likely change most of the times. What do you think?

Reply:
Do we have a non-staging endpoint available for routing tiles?

Reply:
Yeah I think the driver app is a good place to let our unimplemented responsibilities live. I don't think we would give an SDK to a customer with these defaults, would we? If we would, maybe I don't have much to be concerned about.

I think NN would be blocked by routing tiles not loading though. So am happy to move forward and fix this to unblock development

Reply:
@LukasPaczos 
> Do we have a non-staging endpoint available for routing tiles?

Yeah. `https://api.mapbox.com`

Reply:
@kmadsen 
> I don't think we would give an SDK to a customer with these defaults, would we?

Yeah, that's what I was wondering in https://github.com/mapbox/mapbox-navigation-android/pull/2480#discussion_r380300652 and I don't think we would either.

> I think NN would be blocked by routing tiles not loading though. So am happy to move forward and fix this to unblock development

What do you mean? If you're referring to `I'm currently running into an issue on my device, where tiles in the free-drive mode are not being pre-fetched.` that was fixed in https://github.com/mapbox/mapbox-navigation-android/blob/e78b5062585959753e5debfaecbbd1e3f9256097/libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/MapboxOnboardRouter.kt#L65 https://github.com/mapbox/mapbox-navigation-android/pull/2480#issuecomment-586376373 - temporarily using the HTTP client from NN until NN solves the issue they run in their side with Bindgen when using `HttpInterface`. Meaning, that's not blocking this PR 🚀

Reply:
What is the URL for routing tiles at `https://api.mapbox.com`?

Also, we are using the staging endpoint in the legacy SDK - https://github.com/mapbox/mapbox-navigation-android/blob/2afe0e5f47a6a7540acd44741447333595942fa7/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java#L1046-L1048

Can we resolve that TODO now? Otherwise, I think we should merge and address this whenever we have the right endpoint available.

Reply:
> What is the URL for routing tiles at `https://api.mapbox.com`?

`https://api.mapbox.com/route-tiles/v1/` not sure how up to date it is though.

> Also, we are using the staging endpoint in the legacy SDK

Noting that https://github.com/mapbox/mapbox-navigation-android/blob/2afe0e5f47a6a7540acd44741447333595942fa7/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java#L1046-L1048 is _Free Drive_-related (and not _Offline_ in general) and _Free Drive_ has never been released publicly (not available in any of the 0.x releases). We've been testing it internally though, reason why we have those hardcoded values in there.

In any case, I'm 🆗 addressing this in a separate PR but the overall discussion `Do we want to expose these options as default` still remains. Do we? @LukasPaczos @kmadsen

Reply:
> Free Drive has never been released publicly (not available in any of the 0.x releases)

That makes sense, thanks @Guardiola31337 :+1:

In this case, let's do the configuration on the client-side, for now, using the options builder. I'm going to add this test configuration for the example activity instead.